scss/_off.scss

```diff
@@ -721,52 +731,96 @@ html[dir="rtl"] {
 // as we use them only for knowledge panel, with only 1 openable element
 // per accordion
 .accordion .accordion-navigation>a, .accordion dd>a {
-  background:#fafafa;
+  background:#ffffff;
+  border-radius:8px;
 }
 .accordion .accordion-navigation.active>a, .accordion dd.active>a  {
-  background:#fafafa;
+  background:#ffffff;
 }
 .accordion .accordion-navigation>a:hover, .accordion dd>a:hover {
   background:#f0f0f0;
+  text-decoration:none;
 }
 
 // background colors for attributes and panels
+
+.evaluation_good_title {
+  color:rgba(33,150,83,1);
```

Comment:
As we use scss why don't we use variables for those things ?

Reply:
Just because I'm not used to it. Changed to scss variables.